virttest/test_setup.py

```diff
@@ -868,6 +868,8 @@ def _enable_nat(self):
             self._iptables_add(rule)
 
     def _start_dhcp_server(self):
+        if not utils_package.package_install("dnsmasq"):
+            exceptions.TestSkipError("Failed to install dnsmasq package")
         process.system("service dnsmasq stop")
```

Comment:
let's stop dnsmasq service with service module in framework  to make code works more stable. eg, 
`
 service.SpecificServiceManager("dnsmasq").stop()`

Other parts looks good to me.

Reply:
> let's stop dnsmasq service with service module in framework to make code works more stable. eg,
> ` service.SpecificServiceManager("dnsmasq").stop()`
> 
Updated.
> Other parts looks good to me.

Also, I replaced some steps with the Bridge Manager method, test pass with `netdst=private` and `testcase=boot`. Please help to review again, thanks.